composer.json

```diff
@@ -65,6 +65,7 @@
         "symfony/dependency-injection": "4.4.37",
         "symfony/event-dispatcher": "4.4.37",
         "symfony/event-dispatcher-contracts": "1.1.11",
+        "symfony/http-client": "5.4.8",
```

Comment:
symfony version should be same 4.4.37 as other (this is a good reminder that in the future before 11/2022 we will need to upgrade symfony packages to 5.4 version)